client/src/main/scala/org/zalando/nakadi/client/utils/ClientBuilder.scala

```diff
@@ -7,26 +7,33 @@ import org.zalando.nakadi.client.scala.ClientImpl
 import org.zalando.nakadi.client.scala.Connection
 import java.util.function.Supplier
 import com.google.common.base.Preconditions._
+import org.slf4j.LoggerFactory
 
 object ClientBuilder {
 
   def apply(host: String = null,
-            port: Int = DEFAULT_PORT,
+            port: Int = DEFAULT_PORT.toInt,
             tokenProvider: Option[() => String] = None,
             securedConnection: Boolean = true,
             verifySSlCertificate: Boolean = true) =
     new ClientBuilder(host, port, tokenProvider, securedConnection, verifySSlCertificate)
 
-  private val DEFAULT_PORT = 443
+  private val DEFAULT_PORT = "443"
```

Comment:
why did you change the type from Int to String? If I see it correctly, you always call DEFAULT_PORT.toInt anyway